### ISSUE TITLE
[Feature Request] Only show search on explore page

### ISSUE THREAD

**Issue by Xyl-AU** [Dr-TSNG/TwiFucker#83]
### Is your feature request related to a problem?/你的请求是否与某个问题相关？

I sometimes want to search Twitter, but I don't want to see trends or any of the other outrage-baiting algorithm garbage. Unfortunately, the only way to search is to use the explore page.

### Describe the solution you'd like/描述你想要的解决方案

I'd like an option to only display the search bar at the top on the explore page.

### Additional context/其他信息

Below is a screenshot of my mobile browser Twitter using an adblocker with filters to block those elements. I'd like it to basically look like this in the app.

![Screenshot_20221222_143421_Firefox Nightly](https://user-images.githubusercontent.com/100065463/209057122-feb1c089-c70b-437e-b99c-cf77cc8be6a8.png)

**Comment by A-of-Wonder** [comment]
I agree with this feature request. Especially recently, trends have gotten really awful.
